### PR TITLE
B-19835 and B-19836 TRDM tests

### PR DESCRIPTION
**Pull request by TevinAdams**
## Summary

This merge request introduces additions of test_db credentials to the .envrc and tests for insertTransportaQonAccounQngCodes and insertLinesOfAccounting. Instructions to run the tests have also been added to the readMe.

## Testing
1. In mymove repo run `make db_test_create`
2. Then in mymove repo run `make db_test_migrate_standalone`

### CODE REVIEW

**Comment by cameroncaci** [comment]
Build is failing because it is missing a GH token

**Review by cameroncaci — CHANGES_REQUESTED** [review]
2 tests failing during build

File: scripts/build [hunk 1]
```diff
@@ -6,7 +6,8 @@ set -eu -o pipefail
 echo "Packaging..."
 
 # Build via sam
-sam build
+# sam build --no-tests
+MAVEN_OPTS="-DskipTests=true" sam build
```
Comment (cameroncaci) [inline scripts/build:10]: Should we mark this with an issue prior to merging so we can ensure this is tracked? The fix to reimplement the tests would go back under this feature https://www13.v1host.com/USTRANSCOM38/Epic.mvc/Summary?oidToken=Epic%3A918394

File: README.md [hunk 2]
```diff
@@ -174,10 +178,19 @@ When adding custom properties to the application you can add them directly the t
...
 [Go here](docs/getTable.md)
+
+### Running Tests
+
+Before running tests first run the following commands in the mymove repo to set up the test_db if it is not already set up.:
+- In mymove repo
+1. make db_test_create
+2. make db_test_migrate_standalone
+
+After running the commands mentioned above in the mymove repo, run `mvn test` in the trdm-lambda repo to run the tests.
```
Comment (cameroncaci) [inline README.md:196]: Duplicate of line 30

**Review by cameroncaci — CHANGES_REQUESTED** [review]

File: scripts/build (hunk 1 above)
Comment (TevinAdams) [reply scripts/build:10]: Working on this now.

File: README.md (hunk 2 above)
Comment (TevinAdams) [reply README.md:196]: Just noticed that. Removing my edits.

Comment (TevinAdams) [reply README.md]: This read me edit has been removed.

**Comment by TevinAdams** [comment]
> Release will fail because it is currently missing a GH token

Added the new GH Token.

**Review by cameroncaci — APPROVED** [review]
Beautiful work

**Review by deandreJones — APPROVED** [review]

**Comment by TevinAdams** [comment]
> @TevinAdams Please see this article for triggering the release pipeline with a correct version https://dp3.atlassian.net/wiki/spaces/MT/pages/2486796292/How+to+trigger+and+version+release

Running through these steps now.

**Pull request merged**